recipe/deploy/writable.php

```diff
@@ -24,7 +24,7 @@
 
     if ($httpUser === false && $mode !== 'chmod') {
         // Detect http user in process list.
-        $httpUser = run("ps axo user,comm | grep -E '[a]pache|[h]ttpd|[_]www|[w]ww-data|[n]ginx' | grep -v root | head -1 | cut -d ' ' -f1");
+        $httpUser = run("ps axo comm,user | grep -E '[a]pache|[h]ttpd|[_]www|[w]ww-data|[n]ginx' | grep -v root | head -1 | cut -d ' ' -f2");
```

Comment:
So be here:
```
deployer@app:~/core/current$ ps axo comm,user | grep -E '[a]pache|[h]ttpd|[_]www|[w]ww-data|[n]ginx' | grep -v root | head -1 | cut -d ' ' -f2

deployer@app:~/core/current$ ps axo user,comm | grep -E '[a]pache|[h]ttpd|[_]www|[w]ww-data|[n]ginx' | grep -v root | head -1 | cut -d ' ' -f1
www-data
```

First call is new command. Second all old one.

Reply:
@antonmedv Thank you to find my wrong code. I fixed what you said.